test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -4,12 +4,28 @@
 import static org.junit.Assert.assertTrue;
 
 import java.util.Arrays;
-import java.util.List;
 
 import org.junit.Test;
 
 public class UtilsTest {
 
+    @Test
+    public void isAnyNull() throws Exception {
```

Comment:
You may consider segregating the different methods into its respective categories e.g.
```
// Empty list
....

// Non-empty list with null
...

// Non-empty list without null
...
```
As compared to your current implementation, it would improve readability for yourself and others should there be a need to include more test cases in the future :)